app/renderer/js/utils/link-util.ts

```diff
@@ -19,24 +19,24 @@ export async function openBrowser(url: URL): Promise<void> {
 		);
 		const file = path.join(dir, 'redirect.html');
 		fs.writeFileSync(file, `\
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="UTF-8" />
-        <meta http-equiv="Refresh" content="0; url=${escape(url.href)}" />
-        <title>Redirecting</title>
-        <style>
-            html {
-                font-family: menu, "Helvetica Neue", sans-serif;
-            }
-        </style>
-    </head>
-    <body>
-        <p>Opening <a href="${escape(url.href)}">${escape(url.href)}</a>…</p>
-    </body>
-</html>
-`);
-		shell.openItem(file);
+		<!DOCTYPE html>
+		<html>
+			<head>
+				<meta charset="UTF-8" />
+				<meta http-equiv="Refresh" content="0; url=${escape(url.href)}" />
+				<title>Redirecting</title>
+				<style>
+					html {
+						font-family: menu, "Helvetica Neue", sans-serif;
+					}
+				</style>
+			</head>
+			<body>
+				<p>Opening <a href="${escape(url.href)}">${escape(url.href)}</a>…</p>
+			</body>
+		</html>
+		`);
```

Comment:
The lack of indentation was intentional. Any extra whitespace would literally be written to the file. Whitespace before `<!DOCTYPE html>` is incompatible with at least some old web browsers. I’m not sure if it matters to any current ones. But this question is, at the very least, totally unrelated to the purpose of this commit, so don’t change the indentation here.